Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+use Symfony\CS\Fixer;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Test that parses and runs the fixture *.test files found in
+ * /Fixtures/Integration.
+ *
+ * @author SpacePossum
+ */
+class IntegrationTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Fixer
+     */
+    private $fixer;
+
+    /**
+     * @dataProvider getTests
+     */
+    public function testIntegration($fileName, $testTitle, $fixers, $input, $expected = null)
+    {
+        $this->fixer = new Fixer(); // cannot be done in setUp(): https://github.com/sebastianbergmann/phpunit/issues/836
```

Comment:
the issue you link has nothing to do with setUp. It talks about setUpBeforeClass. So the comment is wrong